Kernel/Output/HTML/Templates/Standard/AdminSystemConfiguration.tt

```diff
@@ -9,7 +9,7 @@
 
 [%
     SET DocumentationLinkText  = Translate('online administrator documentation');
-    SET DocumentationLink      = "<a href='https://doc.otrs.com/doc/manual/admin/" _ Data.ManualVersion _ ".0/en/html/configuration.html#sysconfig' target='_blank'>" _ DocumentationLinkText.html() _ "</a>";
+    SET DocumentationLink      = "<a href='https://doc.znuny.org/doc/manual/admin/" _ Data.ManualVersion _ ".0/en/html/configuration.html#sysconfig' target='_blank'>" _ DocumentationLinkText.html() _ "</a>";
```

Comment:
@tipue-dev Here it should be https://doc.znuny.org/manual/admin/

Reply:
Do you mean the link should only be ```https://doc.znuny.org/manual/admin/ ```? Because i can't find a page describing the system configuration in this version of the manual.